client_cli/src/main.rs

```diff
@@ -515,7 +515,13 @@ mod domain {
             /// A key of metadata
             #[arg(short, long)]
             key: Name,
-            /// A value of metadata
+            /// The value of the metadata to be associated with a specific key.
+            /// The following types are supported:
+            /// Numbers: with a suffix, e.g. 42_u32 or 1000_u128
+            /// Booleans: false/true
+            /// IPv4/IPv6: e.g. 127.0.0.1, ::1
+            /// Iroha Public Key Multihash: e.g. ed01207233BFC89DCBD68C19FDE6CE6158225298EC1131B6A130D1AEB454C1AB5183C0
+            /// JSON: e.g. {"Vec":[{"String":"a"},{"String":"b"}]}
```

Comment:
Isn't it possible to re-use doccomments? It really seems easy to miss when such stuff is updated. 

Maybe use a similar approach as here?

https://github.com/hyperledger/iroha/blob/925b556e31afd15fb975aabf6b80746d0b9a59c4/client_cli/src/main.rs#L26-L35